app/listeners/organization.created/createAdmin.js

```diff
@@ -0,0 +1,22 @@
+var blueprint = require ('@onehilltech/blueprint')
+  , mongodb = require ('@onehilltech/blueprint-mongodb')
+  , ResourceController = mongodb.ResourceController
+  , users   = require ('../../../tests/fixtures/users')
+  ;
+
+var User = require ('../../models/User')
+  ;
+
+module.exports = exports = function (organization) {
+
+  adminData = users[2];
```

Comment:
I would not use test data in the actual source code, I would create a generic admin json object based on the information from the organization and some default values.

example:
```
var adminData = {
  email: 'admin@organization.com', // ????
  username: 'admin',
  password: 'password',
  org_id: organization._id,
  job_title: 'administrator',
  role: 'admin'
}
```
I am still a little unsure about how we are going to get the email to this point.